### PR TITLE
ci: Upgrade to use codecov-action/v1.0.6

### PR DESCRIPTION
**Pull request by FabioLuporini**

### CODE REVIEW

**Comment by mloubout** [comment]
I think coverage is fixed with nbval too so would use the occasion to remove it from requirements and put back coverge of tutos

File: .github/workflows/tutorials.yml [hunk 1]
```diff
@@ -90,20 +90,28 @@ jobs:
 
     - name: Seismic notebooks
       run: |
-        $RUN_CMD py.test --nbval $SKIP examples/seismic/tutorials/
+        $RUN_CMD py.test --nbval --cov . --cov-config=.coveragerc --cov-report=xml:seis_coverage.xml $SKIP examples/seismic/tutorials/
```
Comment (FabioLuporini) [inline .github/workflows/tutorials.yml]: out of curiosity, what are the extra options for?

Comment (mloubout) [reply .github/workflows/tutorials.yml]: Same as in `pytest` one, basically config, report format and name

**Comment by FabioLuporini** [comment]
Merged.

**Pull request merged**